sufia-models/app/models/datastreams/fits_datastream.rb

```diff
@@ -144,42 +144,7 @@ def self.xml_template
     http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd",
                version: "0.6.0",
                timestamp: "1/25/12 11:04 AM") {
-        xml.identification {
```

Comment:
How is this change relevant to the goal of the PR? This seems like a backwards incompatible change and we should deprecate all these methods.